CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+
+For a primer on our codebase, see ["Repository Contents"](https://github.com/microsoft/react-native-windows/wiki/Repository-Contents) and ["Project Structure."](https://github.com/microsoft/react-native-windows/blob/main/docs/project-structure.md)
+
+---
+
+## 👨‍💻Development
+
+### 🚧 Fork, Clone, Branch and Create your PR
+
+RNW uses the ["Fork and pull"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models#fork-and-pull-model) model for accepting and integrating code changes.
+
+Here's what the contribution process looks like:
+
+1. Fork the repo if you haven't already.
+1. Clone your fork locally.
+1. Create & push a feature branch.
+1. Work on your changes.
+1. Test your changes.
+1. Open a Pull Request (PR) and address any feedback.
+
+> 👉 If this is your first time working with forks, you may find [GitHub's reference on forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) helpful. If this is your first time working with git or GitHub altogether, see the [GitHub Quickstart guide](https://docs.github.com/en/get-started/quickstart) for a walkthrough of setting up git, contributing to projects, and explanations of commonly used terms. 
+
+> ⚛️ For RNW-specific walkthrough on setting up your branch, see the [Setup page](https://github.com/microsoft/react-native-windows/wiki/Setup) in our Wiki. 
+
+### 🧪 Testing
+
+Build and see if your solution works. Consult [Building React-Native-Windows](https://github.com/microsoft/react-native-windows/blob/main/docs/building-rnw.md) if you have problems.
+
+Testing is a key component in the development workflow. If your changes affect existing test cases, or you're working on brand new features and also the accompanying test cases, see [End-to-End Testing](https://github.com/microsoft/react-native-windows/blob/main/docs/e2e-testing.md) for more information about how to validate your work locally.
+
+### ✅ Code Review
+
+When you'd like the team to review your PR, (even if the work is not yet fully-complete), open a PR so that the team can review your work and provide comments, suggestions, and request changes. It may take several cycles, but the end result will be solid, testable, conformant code that is safe for us to merge.
+
+### 🥳 Merge
+
+Once your code has been reviewed and approved by the team, it will be merged into the main branch. Once merged, your PR will be automatically closed. 
+
+---
+
+## 🦸 Thank you
+
+Thank you in advance for your contribution! Now, [what's next on the list](https://github.com/microsoft/react-native-windows/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)? 😜
```

Comment:
Great document, thank you!! 
One slight nitpick, maybe we should add to the development setting that it requires a windows machine in this doc. It might be obvious to us :)

Reply:
good call, added a "System Requirements" callout under Development section.